### ISSUE TITLE
custom creation of projection no longer viable

### ISSUE THREAD

**Issue by MartinBeal** [BirdLifeInternational/track2kba#31]
In `projectTracks,` we set a custom-centered lambert equal-area projection based on the centroid lat lons and do so by specifying the lat lons directly in a proj4string. This way of specifying coordinate systems is being phased out, as discussed [here](https://github.com/ropensci/spatsoc/issues/30), [here](https://github.com/r-spatial/discuss/issues/28) and [here](https://www.r-spatial.org/r/2020/03/17/wkt.html) among other places. 

Problem is, I haven't the faintest idea how to do this within the new coordinate system specification paradigm, especially since this is very much still in development, so not particularly well-documented.

**Comment by MartinBeal** [comment]
This seemed to work itself out once I upgraded to a more recent version of 'sf', which I think also downloaded a version of the PROJ library. I've added a sf version dependency, but not sure if this will ensure the right version of PROJ is available or not. Especially unsure about how this will work on [macs](https://community.rstudio.com/t/how-to-fix-configure-error-libproj-not-found-in-standard-or-given-locations-when-installing-sf-on-macos/58787/21).
